src/SharedComponents/AstroVirtualAssistant/CommandMessageProcessor.tsx

```diff
@@ -5,6 +5,8 @@ import { feedbackCommandProcessor } from './CommandProcessor/FeedbackCommandProc
 import { thumbsCommandProcessor } from './CommandProcessor/ThumbsCommandProcessor';
 import { manageOrg2FaCommandProcessor } from './CommandProcessor/ManageOrg2FaProcessor';
 
+const CONSOLE_TOUR_ID = '60TJ9PZKMXQ9tDS-WC6bMr46C-U';
```

Comment:
Is this the same for prod and stage?

Reply:
Yup! Pendo wasn't set up to care about that. They have a preview environment and thats about it.

https://app.pendo.io/s/5300167311360000/guides/60TJ9PZKMXQ9tDS-WC6bMr46C-U?view=settings